tests/nightly/dist_sync_kvstore.py

```diff
@@ -388,8 +388,42 @@ def check_trainer_kv_type(stype, grad_stype, update_on_kv):
     check_trainer_kv_type('default', 'default', False)
     check_trainer_kv_type('default', 'row_sparse', True)
     check_trainer_kv_type('row_sparse', 'row_sparse', True)
-    print('worker ' + str(my_rank) + ' passed test_gluon_trainer')
-
+    print('worker ' + str(my_rank) + ' passed test_gluon_trainer_type')
+
+def test_gluon_trainer_step():
+    def check_trainer_step():
+        ctx = mx.cpu(0)
```

Comment:
is this on purpose? We run our KVstore tests on a GPU instance. if we don't require GPU, please downgrade to using a cpu instance.